Microsoft.Toolkit.Uwp.SampleApp/SamplePages/DropShadow/DropShadowPage.xaml

```diff
@@ -0,0 +1,104 @@
+                                justo, aliquet a risus ut, condimentum malesuada metus. Duis vehicula pharetra dolor vel finibus. Nunc auctor tortor nunc, in varius velit
+                                lobortis vel. Duis viverra, ante id mollis mattis, sem mauris ullamcorper dolor, sed rhoncus est erat eget ligula. Aliquam rutrum velit et
+                                felis sollicitudin, eget dapibus dui accumsan.
+                    </TextBlock>
+                </controls:CompositionShadow>
+
+                <TextBlock Style="{StaticResource TitleText}" >Shapes</TextBlock>
+                <Border Style="{StaticResource DividingBorder}"/>
+
+                <StackPanel Orientation="Horizontal">
+                    <controls:CompositionShadow BlurRadius="{Binding BlurRadius.Value, Mode=OneWay}" 
+                                                ShadowOpacity="{Binding Opacity.Value, Mode=OneWay}" 
+                                                OffsetX="{Binding OffsetX.Value, Mode=OneWay}" 
+                                                OffsetY="{Binding OffsetY.Value, Mode=OneWay}" 
+                                                Margin="20">
+                        <Polygon Points="50,0 0,50 50,50"
+                            Stroke="BlueViolet" StrokeThickness="2" Fill="Blue"/>
+                    </controls:CompositionShadow>
+
+                    <controls:CompositionShadow BlurRadius="{Binding BlurRadius.Value, Mode=OneWay}" 
+                                                ShadowOpacity="{Binding Opacity.Value, Mode=OneWay}" 
+                                                OffsetX="{Binding OffsetX.Value, Mode=OneWay}" 
+                                                OffsetY="{Binding OffsetY.Value, Mode=OneWay}" 
+                                                Margin="20">
+                        <Rectangle Width="100" Height="50"
+                            Stroke="Green" StrokeThickness="5"/>
+                    </controls:CompositionShadow>
+
+                    <controls:CompositionShadow BlurRadius="{Binding BlurRadius.Value, Mode=OneWay}" 
+                                                ShadowOpacity="{Binding Opacity.Value, Mode=OneWay}" 
+                                                OffsetX="{Binding OffsetX.Value, Mode=OneWay}" 
+                                                OffsetY="{Binding OffsetY.Value, Mode=OneWay}" 
+                                                Margin="20">
+                        <Polyline Points="0,0 50,50 50,0 0,50"
+                            Stroke="Black" StrokeThickness="2"/>
+                    </controls:CompositionShadow>
+                </StackPanel>
+
+                <TextBlock Style="{StaticResource TitleText}" >Images</TextBlock>
+                <Border Style="{StaticResource DividingBorder}"/>
+
+                <StackPanel Orientation="Horizontal">
+                    <controls:CompositionShadow BlurRadius="{Binding BlurRadius.Value, Mode=OneWay}" 
```

Comment:
Good point. Composition use already a class named DropShadow (in Windows.UI.Composition); I just wanted to avoid to have the same name than an existing component. But yes, I can rename it to DropShadow or DropShadowControl if you prefer.